bazel/repository_locations.bzl

```diff
@@ -1056,6 +1035,29 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         sha256 = "ce7a5c76e8b425aca874cea329fd9ac44b203b777053453b6a37b4496c5ce34f",
         strip_prefix = "emsdk-{version}",
         urls = ["https://github.com/emscripten-core/emsdk/archive/{version}.tar.gz"],
+        patch_cmds = [
```

Comment:
nit: Add something like
```
FIXME: https://issues.redhat.com/browse/OSSM-781
```

Ideally we would need to verify if 2.0.7 is still the version used by envoy 1.20. This was true for 1.17.

Reply:
Commented, and will follow up on the issue.